leetcode/add_two_numbers/step3.py

```diff
@@ -0,0 +1,20 @@
+from typing import Optional
+
+class Solution:
+    def addTwoNumbers(self, l1: Optional[ListNode], l2: Optional[ListNode]): Optional[ListNode]:
+        dummy = ListNode()
+        node = dummy
+        carry = 0
```

Comment:
引数のところが矢印ではなく、コロンになってしまっています。

```suggestion
    def addTwoNumbers(self, l1: Optional[ListNode], l2: Optional[ListNode]) -> Optional[ListNode]:
        dummy = ListNode()
        node = dummy
        carry = 0
```

Reply:
指摘ありがとうございます。  
すみません色々とミスってしまっていますね...